modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
I checked on UAT  
I don't think it'd break anything, but if we do get feedback saying "this used to be 54 and now it's 51" this module will be here to make the switchover easy